src/applications/vaos/components/FacilityHours.jsx

```diff
@@ -10,14 +10,22 @@ const FacilityHours = ({ location }) => {
   // Derive the formatted hours info.
   const hoursInfo = location?.hours;
 
-  // Derive the time ranges for each day.
-  const sunday = formatOperatingHours(hoursInfo?.sunday);
-  const monday = formatOperatingHours(hoursInfo?.monday);
-  const tuesday = formatOperatingHours(hoursInfo?.tuesday);
-  const wednesday = formatOperatingHours(hoursInfo?.wednesday);
-  const thursday = formatOperatingHours(hoursInfo?.thursday);
-  const friday = formatOperatingHours(hoursInfo?.friday);
-  const saturday = formatOperatingHours(hoursInfo?.saturday);
+  const days = [
+    'sunday',
+    'monday',
+    'tuesday',
+    'wednesday',
+    'thursday',
+    'friday',
+    'saturday',
+  ];
+
+  const dayObjects = days
+    .map(d => ({
+      day: `${d.charAt(0).toUpperCase()}${d.slice(1)}`,
```

Comment:
It might be cleaner to have the days be capitalized in the `days` array, and call `toLowerCase` to use them as the property for `hoursInfo`.